plugins/koterm.koplugin/aliases.lua

```diff
@@ -0,0 +1,169 @@
+local KeyValuePage = require("ui/widget/keyvaluepage")
+local InfoMessage = require("ui/widget/infomessage")
+local MultiInputDialog = require("ui/widget/multiinputdialog")
+local UIManager = require("ui/uimanager")
+local _ = require("gettext")
+local util = require("util")
+local T = require("ffi/util").template
+
+local Aliases = {
+    filename = "",
+    close_callback = nil,
+    parent = nil,
+
+    alias_kv = nil,
+    kv_pairs = {},
+}
+
+function Aliases:show(filename, close_callback, parent)
+    self.filename = filename
+    self.close_callback = close_callback
+    self.parent = parent
+    self:load()
+
+    self.alias_kv = KeyValuePage:new{
+        title = "Aliases (Shortcuts)",
+        kv_pairs = self.kv_pairs,
+        close_callback = self.close_callback,
+    }
+    UIManager:show(self.alias_kv)
+end
+
+function Aliases:updateKeyValues()
+    self.alias_kv.kv_pairs = self.kv_pairs
+    UIManager:close(self.alias_kv)
+    self.alias_kv = KeyValuePage:new{
+        title = "Aliases (Shortcuts)",
+        kv_pairs = self.kv_pairs,
+        close_callback = self.close_callback,
+    }
+    UIManager:show(self.alias_kv)
+end
+
+function Aliases:load()
+    local file = io.open(self.filename, "r")
+    self.kv_pairs = {}
+    if file then
+        for line in file:lines() do
+            line = line:gsub("^ *alias *", "") -- drop alias
+            local dummy, separator = line:find("^[%a%d][%a%d-_]*%=") -- find separator
+            if line ~= "" and line:sub(1, 1) ~= "#" and separator then
+                local alias_name = line:sub(1, separator - 1)
+                local alias_command = line:sub(separator + 1):gsub("\"", "")
+                table.insert(self.kv_pairs, {alias_name, alias_command,
+                        callback = function() self.editAlias(self, alias_name, alias_command) end
+                })
+            end
+        end
+        file:close()
+        table.sort(self.kv_pairs, function(a, b) return a[1] < b[1] end)
+    end
+
+    table.insert(self.kv_pairs, 1,
+        { _("Tap here to create a new alias"), "", callback = function() self.editAlias(self, "", "") end})
```

Comment:
`Tap to create a new alias` or juste `Create a new alias` ?
(Also used below in some other place.)